src/Controller/ResetPasswordController.php

```diff
@@ -35,33 +33,36 @@ public function __construct(ResetPasswordHelperInterface $resetPasswordHelper)
     }
 
     /**
-     * @Route("/request", name="app_forgot_password_request")
+     * @Route(name="app_forgot_password_request")
```

Comment:
```suggestion
     * @Route("/", name="app_forgot_password_request")
```

I like this because it's a bit more explicit. This one made me think "Hey! What's the path???" :)

Reply:
good point - fixed

Reply:
using `@Route("", ...)` as it doesn't require a trailing slash in url